pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/AuthorizationService.java

```diff
@@ -293,7 +293,7 @@ public CompletableFuture<Boolean> canLookupAsync(TopicName topicName, String rol
             if (isSuperUser) {
                 return CompletableFuture.completedFuture(true);
             } else {
-                return provider.canLookupAsync(topicName, role, authenticationData);
+                return provider.allowTopicOperationAsync(topicName, role, TopicOperation.LOOKUP, authenticationData);
```

Comment:
Please remove the superuser check, because the `provider.allowTopicOperationAsync` includes that.

Reply:
Updated.